HW1/P5/P5_bfs.py

```diff
@@ -0,0 +1,102 @@
+from pyspark import AccumulatorParam
+
+# Accumulator used for BFS. Basically the algorithm adds all the visited
+# nodes in one iteration into this accumulator instance.
+class SetAccm(AccumulatorParam):
```

Comment:
Using the accumulator to store a set of all the nodes and their distances, makes it a very large accumulator, which is not very efficient.